gnomad_qc/v4/annotations/generate_freq.py

```diff
@@ -290,92 +322,105 @@ def generate_freq_and_hists_ht(
         - fixed_homalt_model
         - gatk_version
         - age
-        - sample_age_bin
         - ukb_sample
-        - downsampling
-        - downsamplings
 
     :param vds: Input VDS.
     :param ab_cutoff: Allele balance cutoff to use for high AB het annotation.
-    :param idx: Optional index to append to temp file name.
     :return: Hail Table with frequency and histogram annotations.
     """
     final_rows_anns = {}
     final_globals_anns = {}
 
-    logger.info("Densifying VDS # %s...", idx)
-    mt = hl.vds.to_dense_mt(vds)
-
-    logger.info("Computing sex adjusted genotypes...")
-    mt = mt.transmute_entries(
-        GT=adjusted_sex_ploidy_expr(mt.locus, mt.GT, mt.sex_karyotype),
-    )
-    mt = mt.annotate_entries(adj=get_adj_expr(mt.GT, mt.GQ, mt.DP, mt.AD))
-
-    logger.info("Annotating frequencies and counting high AB het calls...")
+    ht = vds.variant_data.cols()
     additional_strata_expr = [
-        {"gatk_version": mt.gatk_version},
-        {"gatk_version": mt.gatk_version, "pop": mt.pop},
-        # TODO: confirm this is how we want to name this.
-        {"ukb_sample": mt.ukb_sample},
+        {"gatk_version": ht.gatk_version},
+        {"gatk_version": ht.gatk_version, "pop": ht.pop},
+        {"ukb_sample": ht.ukb_sample},
     ]
+    logger.info("Building frequency stratification list..."
```

Comment:
```suggestion
    logger.info("Building frequency stratification list...")
```
oops! My bad, this is breaking pylint.